### DISCUSSION TITLE
Gherkin support.

### DISCUSSION THREAD

**Discussion by furrm** [pester/Pester#2127] [Q&A] [answered]
Hi all.  I am using Pester v4.6.0 and making use of the Invoke-Gherking command.  While I find the support for Gherkin testing to be a great feature, I have noticed that this command is no longer available for v5.  I'm concerned that you may have stopped support so can you confirm whether Gherkin tests continue to be available from v5 onwards?  

Many thanks, Mike

**Comment by fflaten** [comment, marked as answer]
Hi. Gherkin-support was removed in Pester v5. See [breaking changes in v5](https://pester.dev/docs/migrations/breaking-changes-in-v5#actual-breaking-changes)

> Gherkin is removed, and will later move to it's own module, please keep using Pester version 4.

**Comment by PPadgett** [comment]
Any updates, on the Gherkin module or bring Invoke-Gherkin back into pester?  

https://pester.dev/docs/migrations/breaking-changes-in-v5
- "Gherkin is removed, and will later move to it's own module, please keep using Pester version 4."

**Reply by fflaten** [reply]
Nobody's working on it AFAIK. It would be a standalone project created by someone else with dependency on Pester, similar to dbachecks and Maester.
